dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/deduplication/DeduplicationControllerTest.java

```diff
@@ -66,38 +70,37 @@ class DeduplicationControllerTest extends H2ControllerIntegrationTestBase {
   private OrganisationUnit orgUnit;
   private TrackedEntityType trackedEntityType;
   private TrackedEntity origin;
-  private TrackedEntity duplicate1;
+  private TrackedEntity duplicate;
   private PotentialDuplicate potentialDuplicate1;
   private PotentialDuplicate potentialDuplicate2;
+  private User user;
 
   @BeforeEach
   void setUp() {
+    injectAdminIntoSecurityContext();
```

Comment:
Are you doing that to undo switching to another user? Asking because that should already be done for you in https://github.com/dhis2/dhis2-core/blob/5f32ce23c66a2b75018a2cba07529f58e998710e/dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/test/webapi/ControllerIntegrationTestBase.java#L118-L119 

There are so many different `inject/switch/yyy/user` methods that we don't really know which one we should be using but it would be great to figure it out and settle on one 😅

Reply:
good to know thanks!